injector/src/test/java/edu/ucr/cs/riple/injector/OnParameterInjectionTest.java

```diff
@@ -274,4 +274,204 @@ public void parameter_nullable_with_annotation() {
+            "       final @Baz Map<String, @Baz Object> m,",
+            "       @Foo(clazz = String.class, value = \"Some argument\") @Baz Object o1,",
+            "       @Nullable Object o2,",
+            "       Object o3,",
+            "       @Foo(clazz = Object.class, value = \"Some other argument\") @Baz Object o4) {",
+            "     return new @Baz Runnable() {",
+            "       public void run() {",
+            "         System.out.print(\"log\");",
+            "       }",
+            "     };",
+            "   }",
+            "}")
+        .addChanges(
+            new AddMarkerAnnotation(
+                new OnParameter(
+                    "Main.java",
+                    "edu.ucr.Main",
+                    "foo6(Map<String, Object>, Object, Object, Object, Object)",
+                    2),
+                "javax.annotation.Nullable"))
+        .start();
+  }
+
+  @Test
+  public void methodWithMultipleLineDeclarationFifthParam() {
+    injectorTestHelper
+        .addInput(
+            "Main.java",
+            "package edu.ucr;",
+            "public class Main {",
+            "   @Foo(clazz = String.class, value = \"Some description\")",
+            "   private static @Baz Object foo6(",
+            "       final @Baz Map<String, @Baz Object> m,",
+            "       @Foo(clazz = String.class, value = \"Some argument\") @Baz Object o1,",
+            "       Object o2,",
+            "       Object o3,",
+            "       @Foo(clazz = Object.class, value = \"Some other argument\") @Baz Object o4) {",
+            "     return new @Baz Runnable() {",
+            "       public void run() {",
+            "         System.out.print(\"log\");",
+            "       }",
+            "     };",
+            "   }",
+            "}")
+        .expectOutput(
+            "package edu.ucr;",
+            "import javax.annotation.Nullable;",
+            "public class Main {",
+            "   @Foo(clazz = String.class, value = \"Some description\")",
+            "   private static @Baz Object foo6(",
+            "       final @Baz Map<String, @Baz Object> m,",
+            "       @Foo(clazz = String.class, value = \"Some argument\") @Baz Object o1,",
+            "       Object o2,",
+            "       Object o3,",
+            "       @Nullable @Foo(clazz = Object.class, value = \"Some other argument\") @Baz Object o4) {",
+            "     return new @Baz Runnable() {",
+            "       public void run() {",
+            "         System.out.print(\"log\");",
+            "       }",
+            "     };",
+            "   }",
+            "}")
+        .addChanges(
+            new AddMarkerAnnotation(
+                new OnParameter(
+                    "Main.java",
+                    "edu.ucr.Main",
+                    "foo6(Map<String, Object>, Object, Object, Object, Object)",
+                    4),
+                "javax.annotation.Nullable"))
```

Comment:
Just saw all these tests! Nice. Never mind my comment about missing the checks for the arguments `o2` and `o4`. I still think it's worth checking it works if they are on the same line as the other two arguments.

Reply:
FYI, I removed some of the redundant test cases. I 9ffe4939c994252e0198a25b1178aa9bdc138eac.